integration_test/Makefile

```diff
@@ -252,24 +186,22 @@ pg15:
 reload:
 	docker build -f Dockerfile.test-reload $(DOCKER_BUILD_OPTS)
 	$(call docker_run_cmd,)
-	docker exec --user pganalyze --detach pganalyze-collector-test bash -c "pganalyze-collector 2> /tmp/collector.log"
 	sleep 5
+	docker exec pganalyze-collector-test systemctl start pganalyze-collector
 	docker exec pganalyze-collector-test pganalyze-collector --reload
-	docker exec pganalyze-collector-test cat /tmp/collector.log | grep "Reloading configuration..."
+	docker exec pganalyze-collector-test journalctl -u pganalyze-collector | grep "Reloading configuration..."
 	docker exec pganalyze-collector-test test \! -f /var/lib/pganalyze-collector/state
 	docker exec pganalyze-collector-test bash -c "echo 'db_url = postgres://127.0.0.1/postgres' >> /etc/pganalyze-collector.conf"
-	docker exec --user pganalyze pganalyze-collector-test bash -c "echo '' > /tmp/collector.log"
 	docker exec pganalyze-collector-test pganalyze-collector --reload
 	docker exec pganalyze-collector-test pganalyze-collector --reload
-	docker exec pganalyze-collector-test cat /tmp/collector.log | grep "Reloading configuration..."
```

Comment:
I'm guessing we're dropping this second grep because there's no way for us to reset the journal, so the grep might find the older message instead? In that case, do we still want to keep the reloads right above?

Reply:
Yeah, exactly, it was dropped because its difficult to reset the journal (possible, but messy)

The reloads are still needed because we are checking for the presence of the state file, which, as the comment at the beginning of the target notes, only gets created when you reload once a config is present (and that's why two reloads are needed, and this still tests something useful 😄)

Reply:
Sounds good. Since you commented all the interesting pieces here, there's not much value to keeping the history, so I'll just squash this for clarity and merge it as one commit.